packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -108,6 +108,10 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
 
   /**
    * Boolean value determining whether the player should show the now playing notification.
+   *
+   * @default false
+   * @platform ios
+   * @platrorm android
```

Comment:
As per the [documentation writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms), Android should go first
```suggestion
   * @platform android
   * @platrorm ios
```